### PR TITLE
Fix types of connect-sqlite3 to have Connect parameter

### PR DESCRIPTION
**Pull request by oof2win2**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: [this issue](https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/58757)
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by oof2win2** [comment]
`connect` seems irrelevant to me now, it seems much more like express than something with a store. The typings for it don't seem to have any form of a store. Instead, I believe that `express-session` should be used, since that is the only other package that is used.

**Comment by oof2win2** [comment]
`dtslint` and the CI here seem to battle over which tslint.json should be used and what it should extend from. Just a note to maintainers if they read this.

File: types/connect-sqlite3/index.d.ts [hunk 1]
```diff
@@ -2,169 +2,180 @@
...
-	 */
-	dir?: string;
-
-	/**
-	 * Enables [WAL](https://www.sqlite.org/wal.html) mode (defaults to false)
-	 * @default "false"
-	 */
-	concurrentDB?: string;
+declare namespace connect {
+	interface SQLiteStoreOptions {
```
Comment (DangerBotOSS) [inline types/connect-sqlite3/index.d.ts:11]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Review by bl0ggy — APPROVED** [review]
Tested on my code, it's working both at compile time and runtime

**Comment by oof2win2** [comment]
Ready to merge

**Pull request merged**